scipy/stats/tests/mpmath_distribution.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
+from mpmath import mp
+
+mp.dps = 100  # default in case the user forgets to set it
+
+class Distribution:
+    # Minimalist distribution infrastructure for generating reference values
+    # of distribution functions. No edge case handling.
+
+    @staticmethod
+    def _make_mpf_array(x):
+        shape = np.shape(x)
+        x = np.asarray(x, dtype=np.float64).ravel()
+        return np.asarray([mp.mpf(xi) for xi in x]).reshape(shape)[()]
+
+    def __init__(self, **kwargs):
```

Comment:
Since this is for testing, is there some seeding/rng we could put there? Will matter if we add things like `random` (yes I refuse to call it `rvs` 😆).

Reply:
I don't think we will add `rvs`/`random`. If a distribution has a custom `rvs`, its output cannot be compared against the generic `rvs` we'd implement it. If a distribution relies on the default `rvs`, it does not need to be tested.